cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+YARN containers spawned by a CDAP application use the default container logback, ``logback-container.xml``,
+packaged with CDAP. This ``logback.xml`` does log rotation, and depending on the use case, may be sufficient.
```

Comment:
``logback-container.xml`` has configuration to rotate logs once every day and delete logs older than 14 days.